plugin/plugin-config.json

```diff
@@ -1,3 +0,0 @@
-{
-  "kialiUrl": "https://kiali-istio-system.apps-crc.testing"
```

Comment:
My suggestion here is to just remove this line, but do NOT remove the entire file.

I would think chances are good that at some point in the future, we are going to need some ability to configure the  plugin via a config file. So let's leave the framework in place. Even if the config file is empty for now, let's leave it empty but keep it. So when the time comes when we need to add a new config option, we don't have to do the work all over again to add the config file.

Unless you know for sure we will never need a config file - if we won't ever need a config file, then we can remove it.